xwiki-rendering-macros/xwiki-rendering-macro-message/src/main/java/org/xwiki/rendering/internal/macro/message/AbstractMessageMacro.java

```diff
@@ -100,6 +101,16 @@ public List<Block> execute(BoxMacroParameters parameters, String content, MacroT
             }
             // Enhance the default box with an icon as the first element.
             Block iconBlock = iconProvider.get(iconName);
+            // Provide an accessible name besides this icon
+            // This is the responsibility of the message macro and not the iconProvider which should only provide
+            // icons without any semantics
+            if (iconBlock.getClass() == ImageBlock.class) {
+                iconBlock.setAttribute("alt", iconPrettyName);
+            } else if (!iconPrettyName.equals("")) {
+                Block iconAlternative = new FormatBlock(List.of(new RawBlock(iconPrettyName, Syntax.PLAIN_1_0)),
```

Comment:
That's not how you insert plain text. You need to parse the text using a `plain/1.0` parser and insert the resulting blocks. Raw blocks are only displayed in the respective output syntax, i.e., this would only be displayed when rendering the macro to `plain/1.0` syntax.

Reply:
Addressed in https://github.com/xwiki/xwiki-rendering/pull/296/commits/554112649ff574ae69f7821fbf60a49e5faae5fe :+1: